modules/Calendar/Dashlets/CalendarDashlet/CalendarDashlet.php

```diff
@@ -34,97 +34,135 @@
-		ob_start();
-		
-		if(isset($GLOBALS['cal_strings']))
-			return parent::display() . "Only one Calendar dashlet is allowed.";
-			
-		require_once('modules/Calendar/Calendar.php');
-		require_once('modules/Calendar/CalendarDisplay.php');
-		require_once("modules/Calendar/CalendarGrid.php");
-		
-		global $cal_strings, $current_language;
-		$cal_strings = return_module_language($current_language, 'Calendar');
-		
-		if(!ACLController::checkAccess('Calendar', 'list', true))
-			ACLController::displayNoAccess(true);
-						
-		$cal = new Calendar($this->view);
-		$cal->dashlet = true;
-		$cal->add_activities($GLOBALS['current_user']);
-		$cal->load_activities();
-		
-		$display = new CalendarDisplay($cal,$this->id);
-		$display->display_calendar_header(false);
-
-		$display->display();
-
-		$str = ob_get_contents();
-		ob_end_clean();
-		
-		return parent::display() . $str;
+    /**
+     * @see Dashlet::display()
+     */
+    public function display()
+    {
+        ob_start();
+
+        if (isset($GLOBALS['cal_strings'])) {
+            return parent::display() . "Only one Calendar dashlet is allowed.";
+        }
+
+        require_once('modules/Calendar/Calendar.php');
+        require_once('modules/Calendar/CalendarDisplay.php');
+        require_once("modules/Calendar/CalendarGrid.php");
+
+        global $cal_strings, $current_language;
+        $cal_strings = return_module_language($current_language, 'Calendar');
+
+        if (!ACLController::checkAccess('Calendar', 'list', true)) {
+            ACLController::displayNoAccess(true);
+        }
+
+        $cal = new Calendar($this->view);
+        $cal->dashlet = true;
+        $cal->add_activities($GLOBALS['current_user']);
+        $cal->load_activities();
+
+        $display = new CalendarDisplay($cal, $this->id);
+        $display->display_calendar_header(false);
+
+        $display->display();
+
+        $str = ob_get_contents();
+        ob_end_clean();
+
+        return parent::display() . $str;
     }
-    
 
-    function displayOptions() {
-        global $app_strings,$mod_strings;        
+    /**
+     * @see Dashlet::displayOptions()
+     */
+    public function displayOptions()
+    {
+        global $app_strings, $mod_strings;
         $ss = new Sugar_Smarty();
-        $ss->assign('MOD', $this->dashletStrings);        
+        $ss->assign('MOD', $this->dashletStrings);
         $ss->assign('title', $this->title);
         $ss->assign('view', $this->view);
         $ss->assign('id', $this->id);
 
-        return parent::displayOptions() . $ss->fetch('modules/Calendar/Dashlets/CalendarDashlet/CalendarDashletOptions.tpl');
-    }  
+        return parent::displayOptions() .
+            $ss->fetch('modules/Calendar/Dashlets/CalendarDashlet/CalendarDashletOptions.tpl');
+    }
 
-    function saveOptions($req) {
-        global $sugar_config, $timedate, $current_user, $theme;
+    /**
+     * @see Dashlet::saveOptions()
+     */
+    public function saveOptions($req)
```

Comment:
There seems to be a mixture of $_REQUEST and $req in this function. Are these the same?

The PHP Doc needs to be updated to represent this param. Also if the $req is the same as $_REQUEST, we should  use the param instead of the $_REQUEST.